### PR TITLE
Created the May monthly update

### PR DESCRIPTION
**Pull request by rubymorillo**
May monthly update draft ready for review. Could use assistance with finding a notable contribution from RubyGems.org and RubyGems to feature.

### CODE REVIEW

File: app/views/news/2018-06-07-may-2018-monthly-update.html.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Hello! Welcome to the monthly update. During May, our work was supported by [GitLab](https://about.gitlab.com/), [Stitch Fix](http://multithreaded.stitchfix.com/), and many others.
+
+## ruby together news
+
+In May, Ruby Together was supported by 77 different companies, including Sapphire member [Stripe](https://stripe.com). [Murb](https://murb.nl/) and [StackPath](https://www.stackpath.com/) joined as Onyx members, and developer Gregory Brown (@practicingdev) became a member. 
+
+We were supported by a total of 86 individual members and 68 friends of Ruby Together. Thanks to all of our members for making everything that we do possible.
+
+In community events news, our developer evangelist [PJ Hagerty](http://twitter.com/aspleenic) spoke at MalmoRB and Ruby Ireland, and visited CopenhagenRB among other Meetups.
```
Comment (rubymorillo) [inline app/views/news/2018-06-07-may-2018-monthly-update.html.md]: @indirect do we want to add anything here re: GSoC students?

File: app/views/news/2018-06-07-may-2018-monthly-update.html.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## bundler news
+
+In May, Bundler saw a few bug fixes, the addition of a [new `--skip-install` flag](https://github.com/bundler/bundler/commit/9e87a1ca4b0c3002ac2774e4837234cef7e3ce08) to `bundler add`, and a [new guide on writing Bundler plugins](https://bundler.io/v1.16/guides/bundler_plugins.html). (Thank you [@jules2689](https://github.com/jules2689) for producing this guide!)
+
+Bundler gained 9 new commits, contributed by 3 authors. There were 66 additions and 48 deletions across 17 files.
+
+## rubygems.org news
+
+[Some significant changes in May.] In addition, we [revised the site footer](https://github.com/rubygems/rubygems.org/commit/8de0296d1222e9819ca3a70f678baca0484b99b1) to better clarify how Ruby Central, Ruby Together, and Fastly partnered on RubyGems.org.
```
Comment (rubymorillo) [inline app/views/news/2018-06-07-may-2018-monthly-update.html.md]: >[Some significant changes in May.]

Note: this is just placeholder copy ^^^

File: app/views/news/2018-06-07-may-2018-monthly-update.html.md (hunk 1 above)
Comment (segiddins) [reply app/views/news/2018-06-07-may-2018-monthly-update.html.md]: for bundler, we've made lots of progress on `bundle remove`!

Comment (rubymorillo) [reply app/views/news/2018-06-07-may-2018-monthly-update.html.md]: @segiddins yup, looks like @indirect added that at the top of the update! 😄

**Pull request merged**